### ISSUE TITLE
Ignoring 'background-image' makes PageSpeed Insights report a bunch of above-the-fold optimizations

### ISSUE THREAD

**Issue by rfgamaral** [addyosmani/critical#251]
Take the following Gulp function that I use to extract my critical-path CSS:

```js
function extractCriticalPathCssTask() {
    return critical.generate({
        base: './dist',
        src: 'index.html',
        dest: 'index.html',
        inline: true,
        minify: true,
        ignore: [
            '@font-face',
            'background-image'
        ]
    });
}
```

PageSpeed Insights gives me this:

![image](https://user-images.githubusercontent.com/96476/32404857-3a0c7fac-c151-11e7-9f20-36cdb9925b60.png)

Now, if I remove the `'background-image'` line from the `ignore` options, I get 0 "above-the-fold content" recommendations.

Am I doing something wrong?

**Comment by rfgamaral** [comment]
I did some further debugging on this issue and I don't understand what's going on... PageSpeed splits reports between mobile and desktop, let's focus on mobile for now.

If I have the `critical` rules above, especially the `background-image` one, I get all those warnings. But If I inline just the following code (which got stripped given the `critical` rules):

```css
@media (min-height:481px) and (max-height:736px) and (orientation:portrait) {
    #background {
    background-image: url(assets/images/background-736h.8f5354d0.jpg);
    }
}
```

I no longer get the warnings above, everything is fine. I don't get this... Why do I get all those render-blocking resources (JS and CSS) if I simply remove the inlined `background-image` that is part of the critical-path CSS?

In other words, I want to inline all my CSS but load background images asynchronously, is this not possible? Am I doing something wrong?

**Comment by rfgamaral** [comment]
I think this is has something to do with the `loadCSS` function, it's blocking everything else, somehow.

**Comment by XhmikosR** [comment]
This shouldn't happen any more.
